lib/pharos/phases/configure_master.rb

```diff
@@ -346,6 +346,11 @@ def push_authentication_token_webhook_config
       end
 
       def configure
+        cfg = generate_config
```

Comment:
This probably fixes some other issues as well. Anyway we need/should rewrite control plane config always.

Reply:
This is going to fail for things like enabling the `@config.audit`, unless you also call `push_audit_config` to write those referenced config files as well.